yarn.lock

```diff
@@ -6639,6 +6639,14 @@
     "@typescript-eslint/types" "5.29.0"
     "@typescript-eslint/visitor-keys" "5.29.0"
 
+"@typescript-eslint/scope-manager@5.30.6":
```

Comment:
Why was the lock file changed? Nothing in this PR seems to depend on an old version of @typescript-eslint/*

Changes in this file should be reverted IMHO

Reply:
🤔 right, missed this change. Probably got in after a local yarn install on my machine. I'll revert tomorrow